SDPi_Supplement/asciidoc/volume1/tf1-ch-b-ref-standards-conformance.adoc

```diff
@@ -21,22 +16,55 @@ NOTE: the following section was moved from the mothership document to here for i
 
+* [[[ihe_pcd_tf_1_2019,IHE PCD TF-1:2019]]] IHE Patient Care Device Technical Framework, Vol. 1, available at https://www.ihe.net/uploadedFiles/Documents/PCD/IHE_PCD_TF_Vol1.pdf.
+
+* [[[ihe_pcd_tf_2_2019,IHE PCD TF-2:2019]]] IHE Patient Care Device Technical Framework, Vol. 2, available at https://www.ihe.net/uploadedFiles/Documents/PCD/IHE_PCD_TF_Vol2.pdf.
+
+* [[[ihe_pcd_tf_3_2019,IHE PCD TF-3:2019]]] IHE Patient Care Device Technical Framework, Vol. 3, available at https://www.ihe.net/uploadedFiles/Documents/PCD/IHE_PCD_TF_Vol3.pdf.
+
+* [[[ieee_11073_10702_202x,IEEE 11073-10702:202x]]] P11073-10702™/D1 Draft Standard for Health Informatics – Device Interoperability – Part 10702: Point-of-Care Medical Device Communication – Alert Provisioning by Participants in a Service-Oriented Device Connectivity (SDC) System
+
+* [[[ieee_11073_10701_2022,IEEE 11073-10701:2022]]] P11073-10701™/D4 Draft Standard for Health Informatics – Device Interoperability – Part 10701: Point-of-Care Medical Device Communication - Metric Provisioning by Participants in a Service-Oriented Device Connectivity (SDC) System
+
 * [[[ieee_11073_10700_2022,IEEE 11073-10700:2022]]] P11073-10700™/D7 Draft Standard for Health Informatics – Device Interoperability – Part 10700: Point-of-Care Medical Device Communication – Standard for Base Requirements for Participants in a Service-Oriented Device Connectivity (SDC) System.
 
 * [[[ws_discovery_2009,WS-Discovery:2009]]] Web Services Dynamic Discovery (WS-Discovery) Version 1.1, OASIS Standard, 1 July 2009, available at http://docs.oasis-open.org/ws-dd/discovery/1.1/wsdd-discovery-1.1-spec.html
 
 * [[[dpws_2009,DPWS:2009]]] Devices Profile for Web Services Version 1.1, OASIS Standard, 1 July 2009, available at http://docs.oasis-open.org/ws-dd/dpws/wsdd-dpws-1.1-spec.html
 
-* [[[ws_transfer_2006, WS-Transfer:2006]]] Web Services Transfer (WS-Transfer), W3C Standard, 27 September 2006, available at https://www.w3.org/Submission/WS-Transfer/
+* [[[ws_transfer_2006,WS-Transfer:2006]]] Web Services Transfer (WS-Transfer), W3C Standard, 27 September 2006, available at https://www.w3.org/Submission/WS-Transfer/
+
+* [[[ihe_pcd_sdpi_white_paper_2019,IHE PCD SDPi White Paper:2019]]] IHE Patient Care Devices (PCD) Service-oriented Device Point-of-Care Interoperability (SDPi) White Paper, Revision 1.1, November 1, 2019.  Available at https://profiles.ihe.net/DEV/[profiles.ihe.net/DEV/].
+
+* [[[ihe_pcd_sdpi_use_cases_compendium_2019,IHE PCD SDPi Use Cases Compendium:2019]]] IHE Patient Care Devices (PCD) Compendium of Medical Device Oriented Use Cases, Companion to the "Service-oriented Device Point-of-Care Interoperability (SDPi)" White Paper, Revision 1.1, November 1, 2019.  Available at https://profiles.ihe.net/DEV/[profiles.ihe.net/DEV/].
+
+#TODO: DOES ASCIIDOC NOT SUPPORT REPLACEMENT CHARACTERS (SEE NEXT TWO) OR IS IT DONE IN PUBLICATION TO HTML?#
```

Comment:
Unicode replacement characters? � or ￼?

Reply:
Well [AsciiDoctor "Character Replacement Substitutions"](https://docs.asciidoctor.org/asciidoc/latest/subs/replacements/).  If I remember right, when rendered in the entry it looked fine but when in the reference [[[...]]] ... it didn't handle the superscript directives.  So you got the special character but instead of it being superscripted - like it is subsequently in the block - you just get "<sup> ... </sup>", though the AsciiDoc content is the same.